aea/cli/scaffold.py

```diff
@@ -155,3 +163,44 @@ def _scaffold_item(click_context, item_type, item_name):
         )
     except Exception as e:
         raise click.ClickException(str(e))
+
+
+def _scaffold_dm(click_context):
+    """Add a scaffolded decision maker to the project and configuration."""
+
+    ctx = cast(Context, click_context.obj)
+    existing_dm_path = getattr(ctx.agent_config, "decision_maker_path")
+
+    # check if we already have a decision maker in the project
+    if existing_dm_path is not None:
+        raise click.ClickException("A decision maker file already exists. Aborting...")
+
+    try:
+        agent_name = ctx.agent_config.agent_name
+        click.echo(
+            "Adding decision maker scaffold to the agent '{}'...".format(agent_name)
+        )
+
+        # create the file name
+        dest = Path("decision_maker.py")
+
+        # copy the item package into the agent project.
+        src = Path(os.path.join(AEA_DIR, "decision_maker", "scaffold.py"))
+        logger.debug("Copying decision maker. src={} dst={}".format(src, dest))
+        shutil.copyfile(src, dest)
+
+        # add the item to the configurations.
+        logger.debug(
+            "Registering the decision_maker into {}".format(DEFAULT_AEA_CONFIG_FILE)
+        )
+        import pdb
+
+        pdb.set_trace()
+        ctx.agent_config.decision_maker_path = str(dest)
```

Comment:
I suggest using the following format:

```
path.to.module:ClassName
```

It would give much more flexibility to the developer. That would point to a precise class.

Reply:
this is extendible to other components like protocols, connections, handlers, behaviours, models etc.

It imposes far less restrictions to the name of the classes. Of course, by convention, they could be called `XXXHandler`, but IMHO it shouldn't be a requirement.

Reply:
that being said, I think it's out of the scope of this PR, since it is a more general issue. Thoughts?